docs/networking/cilium.md

```diff
@@ -43,6 +45,15 @@ Make sure `instanceGroup` match the other etcd clusters.
     name: cilium
 ```
 
+If this is an existing cluster, it is important that you roll the entire cluster so that all the nodes can connect to the new etcd cluster.
```

Comment:
```suggestion
If this is an existing cluster, it is important that you perform a rolling update on the entire cluster so that all the nodes can connect to the new etcd cluster.
```